docs/how-to-guides/others/planning-evaluation-using-scenarios.md

```diff
@@ -0,0 +1,297 @@
+execute the scenario.
+
+### Reports
+
+The `Reports` tab displays a list of results from scenario tests. Here, users can view the most recent test executions.
+
+![reports.png](images%2Fplanning-evaluation-using-scenarios%2Freports.png)
+
+The screenshot above illustrates the latest test execution for the public road bus catalog. In the source column, the
+default Autoware repository with the latest commit is displayed. For a deeper analysis, users can click on the title to
+view comprehensive results.
+
+![test-executed-report.png](images%2Fplanning-evaluation-using-scenarios%2Ftest-executed-report.png)
+
+Clicking on the title of a test execution leads to a page displaying build logs and individual test results for each
+suite. By selecting links in the `ID` column, users can access detailed results for each suits.
+
+![suite-test-result.png](images%2Fplanning-evaluation-using-scenarios%2Fsuite-test-result.png)
+
+After clicking the `ID` link, users can view detailed test results for the scenarios within the suite. For instance, in
+our case, 183 out of 198 scenarios succeeded. This page is a useful resource for developers to further investigate
+scenarios that failed.
+
+![single-scenario-report.png](images%2Fplanning-evaluation-using-scenarios%2Fsingle-scenario-report.png)
+
+- `ID` links to a page which shows detailed test results for the scenario.
+- `Scenario Name` links to a page displaying the scenario's definition.
+- `Labels` indicate the conditions under which the scenario failed; if successful, this field remains empty.
+- `Status` shows the count of failed and succeeded cases within a scenario.
+
+To investigate deeper for this specific scenario, we should click into `ID` link.
+
+![report-with-cases.png](images%2Fplanning-evaluation-using-scenarios%2Freport-with-cases.png)
+
+For this scenario, three of the four cases failed. Each case represents changing the `ego_speed` parameter. To deep dive
+into the failed cases, click on the `ID` link.
+
+![single-case-report.png](images%2Fplanning-evaluation-using-scenarios%2Fsingle-case-report.png)
+
+This page show us the detailed information of the failed case. We can understand why the case failing by looking into
+the `Message`. For our case, the message
+is `Simulation failure: CustomCommandAction typed "exitFailure" was triggered by the anonymous Condition (OpenSCENARIO.Storyboard.Story[0].Act["_EndCondition"].ManeuverGroup[0].Maneuver[0].Event[1].StartTrigger.ConditionGroup[1].Condition[0]): The state of StoryboardElement "act_ego_nostop_check" (= completeState) is given state completeState?`
+
+From this message, we can understand that an action which is used for checking the vehicle is not stopped is in complete
+state. And, the scenario author set this condition as a failure condition. Therefore, the scenario failed.
+
+- To test the scenario in local, you can download the scenario and its map by using links above. Running scenario on
```

Comment:
Where are the links above ?

Reply:
Yes I see, I marked them in the image to see easily. Thanks.